R/fullREMIND.R

```diff
@@ -126,6 +126,7 @@ fullREMIND <- function() {
   calcOutput("EEZdistribution",                       round = 4,  file = "p33_EEZdistribution.cs4r")
   calcOutput("ExpertGuess", subtype = "biocharPrices",            file = "p33_BiocharPricePath.cs4r", aggregate = FALSE)
   calcOutput("BiocharBounds",                         round = 2,  file = "p_boundCapBiochar.cs4r")
+  calcOutput("BiocharLimitCropland",                  round = 2,  file = "p33_BiocharLimitCropland.cs4r")
```

Comment:
Currently, `calcBiocharLimitCropland`  is located in mrcommons. So far, it is only used for REMIND input data generation. Will it be used by other mr packages other than mrremind in the future? If not, you should move it to mrremind. mrcommons should only contain functions needed by various packages to avoid multidirectional imports of packages.

Reply:
Thanks, don't forget to delete again mrcommons